edx_dl/parsing.py

```diff
@@ -269,7 +269,8 @@ def extract_unit(self, text, BASE_URL):
         videos = []
         match_metadatas = re_metadata.findall(text)
         for match_metadata in match_metadatas:
-            metadata = json.loads(html_parser.HTMLParser().unescape(match_metadata))
+            metadata = html_parser.HTMLParser().unescape(match_metadata)
+            metadata = json.loads(html_parser.HTMLParser().unescape(metadata))
```

Comment:
Ouch. Unescape, parse and then unescape, parse and then load as json? Well, if it works... :+1: